automatic/awscli/legal/LICENSE.txt

```diff
@@ -0,0 +1,70 @@
+From: https://github.com/aws/aws-cli/blob/develop/LICENSE.txt
```

Comment:
Please follow the same format as used in other packages in this repository.
The mention of where the license has been acquired from should be in `VERIFICATION.txt` file, not in the License file.